sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -685,6 +685,12 @@ def _add_argparse_args(cls, parser):
         help='The time to wait (in milliseconds) for test pipeline to finish. '
              'If it is set to None, it will wait indefinitely until the job '
              'is finished.')
+    # This option is used by Dataflow Runner's Pub/Sub client. Its camelCase
+    # style matches the runner's.
+    parser.add_argument(
+        '--pubsubRootUrl',
```

Comment:
Is there an option to use the standard Python naming here? The lack of consistency will be confusing for users and for testing frameworks.

Reply:
We can perhaps use a python friendly alias. I believe the actual option name will use Java convention because it is used by java runner harness.

Reply:
Done: I think I found an elegant solution.